### ISSUE TITLE
gen3D fails on some molecules

### ISSUE THREAD

**Issue by openbabel-bot** [openbabel/openbabel#1213]
babel -isdf 2Dtonan.sdf -osdf 2Dtonan_3D.sdf --gen3D

The above command used on the attached files results in an SDF with 'nan' in place of coordinates.

Reported by: dhouston6

Original Ticket: [openbabel/bugs/833](https://sourceforge.net/p/openbabel/bugs/833)

**Comment by jamesETsmith** [comment]
Hi! I've run into an issue that I believe is the same except that I'm encountering it in Pybel rather than the command-line interface.

I installed Openbabel/Pybel with Conda and the output of `conda list "openbabel"` is shown below:
```
# packages in environment at /home/james/miniconda3/envs/dbale:
#
# Name                    Version                   Build  Channel
openbabel                 1!2.4.1                  py35_2    omnia
```

I tested all available force fields and got the same error for them all. Here is a minimal example of the error:

```python
import pybel

cases = ["F[Zn]F", "F[Ti](F)F"]
ffs = ["gaff", "ghemical", "mmff94", "mmff94s", "uff"]
for case in cases:
    for ff in ffs:
        m = pybel.readstring("smi", case)
        m.make3D(forcefield=ff)

        print("Force Field =", ff)
        print(m.write("xyz"))
```
```
Force Field = gaff
4

F          1.07585        0.02711        0.05926
Ti         3.24585        0.02711        0.05926
F             -nan           -nan           -nan
F             -nan           -nan           -nan

Force Field = ghemical
4

F          1.02247        0.07565        0.06142
Ti         3.19247        0.07565        0.06142
F             -nan           -nan           -nan
F             -nan           -nan           -nan

Force Field = mmff94
4

F          1.04019        0.07609        0.05095
Ti         3.21019        0.07609        0.05095
F             -nan           -nan           -nan
F             -nan           -nan           -nan

Force Field = mmff94s
4

F          0.97489        0.09279       -0.02756
Ti         3.14489        0.09279       -0.02756
F             -nan           -nan           -nan
F             -nan           -nan           -nan

Force Field = uff
4

F          0.92952       -0.06106       -0.03611
Ti         3.09952       -0.06106       -0.03611
F             -nan           -nan           -nan
F             -nan           -nan           -nan
```

Any advice would be appreciated!

**Comment by ghutchis** [comment]
This is fixed in `master` which will be released on Monday as 3.0.

FYI, most of these force fields don't have parameters for metals. The only thing that will have any effect is UFF.

**Comment by jamesETsmith** [comment]
Great, thanks for the heads up!
